client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-footer.tsx

```diff
@@ -0,0 +1,45 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useTranslate } from 'i18n-calypso';
+import NavigatorHeader from './navigator-header';
+import PatternSelector from './pattern-selector';
+import { useFooterPatterns } from './patterns-data';
+import type { Pattern } from './types';
+
+interface Props {
+	selectedPattern: Pattern | null;
+	onSelect: ( selectedPattern: Pattern ) => void;
+	onBack: () => void;
+	onDoneClick: () => void;
+}
+
+const ScreenHeader = ( { selectedPattern, onSelect, onBack, onDoneClick }: Props ) => {
```

Comment:
The new layout of the "Choose a header"

<img width="212" alt="image" src="https://user-images.githubusercontent.com/13596067/220595827-e774869e-2462-47e2-addd-5467927bde00.png">